source/content/drupal-9.md

```diff
@@ -95,6 +95,49 @@ $node = Node::load(1);
 
 Since most of these changes are relatively minor, there are a number of [deprecation checking and correction tools](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9/deprecation-checking-and-correction-tools) available.
 
+## Test-upgrade an Existing Pantheon Drupal 8 Site to Drupal 9
```

Comment:
I just had another thought that we shouldn't call this "Test-upgrade". Using the word "upgrade" to me sounds like deploying to production. I'd rather have a softer title here, like "Try out Drupal 9 on an Exiting Pantheon site using a Multidev environment".

We can talk about "upgrade" once we have a drops-9 upstream.

Reply:
I tried rewriting this, but the heading without `upgrade` loses the meaning
I think @alexfornuto 's suggestion of the hyphenated version should give folks the idea that this is a trial path. So I'm letting it stand as-is and moving this to copy edit

Reply:
"Test an Existing Pantheon Drupal 8 Site with Drupal 9"?